package.json

```diff
@@ -30,6 +30,7 @@
     "@advanced-rest-client/headers-parser-mixin": "^3.0.0",
     "@advanced-rest-client/response-view": "^3.0.3",
     "@api-components/api-request-editor": "^4.0.0",
+    "@api-components/api-server-selector": "0.0.1",
```

Comment:
If we already putting the selector into other component let's make it at least `0.1.0`. Otherwise with every change in the selector you will have to update all the components that are using this selector.